ports/rp2/mphalport.h

```diff
@@ -68,6 +68,11 @@
 extern int mp_interrupt_char;
 extern ringbuf_t stdin_ringbuf;
 
+#if MICROPY_PY_THREAD
+// Only run the TinyUSB task on CPU 0 to avoid data races
+#define MICROPY_HW_USBD_TASK_CAN_RUN() (get_core_num() == 0)
```

Comment:
@dpgeorge I wasn't sure what the best place for this would be. We could add a mp_usbd_port.h header, but it seems like overkill while there's only a single macro.

Reply:
I think it's fine here.